cclib/bridge/cclib2horton.py

```diff
@@ -53,7 +55,32 @@ def makehorton(ccdat):
     attributes = {}
 
     if hortonver == 2:
-        pass  # Populate with bridge for horton 2 in later PR
+        renameAttrs = {"atomnos": "numbers", "mult": "ms2", "polarizability": "polar"}
+        inputattrs = ccdat.__dict__
+
+        attributes = dict(
+            (renameAttrs[oldKey], val)
+            for (oldKey, val) in inputattrs.items()
+            if (oldKey in renameAttrs)
+        )
+
+        # Rest of attributes need some manipulation in data structure.
+        if hasattr(ccdat, "atomcoords"):
+            # cclib parses the whole history of coordinates in the list, horton keeps the last one.
+            attributes["coordinates"] = ccdat.atomcoords[-1]
+        if hasattr(ccdat, "mocoeffs"):
+            attributes["orb_alpha"] = ccdat.mocoeffs[0]
+            if len(ccdat.mocoeffs) == 2:
+                attributes["orb_beta"] = ccdat.mocoeffs[1]
+        if hasattr(ccdat, "coreelectrons") and isinstance(
+            ccdat.coreelectrons, numpy.ndarray
+        ):  # type-checked
+            attributes["pseudo_numbers"] = ccdat.atomnos - ccdat.coreelectrons
```

Comment:
If you get rid of the type check, you can do
```suggestion
            attributes["pseudo_numbers"] = ccdat.atomnos - np.asanyarray(ccdat.coreelectrons)
```
which will avoid making a copy if `ccdat.coreelectrons` is already an ndarray, but do the conversion if it isn't.

Reply:
@berquist Thanks! I didn't know about `asanyarray`. That definitely simplifies and make it more efficient!